### PR TITLE
enable applying registry settings

### PR DESCRIPTION
**Pull request by MaximilianMeister**
screenshot-20180313112433-1196x370
screenshot-20180313112457-1210x701
screenshot-20180313112527-1206x444

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
[How to apply the changes?](https://trello.com/c/O2BGEKMv/688-how-to-apply-the-changes)

**Review by vitoravelino — DISMISSED** [review]

File: app/models/minion.rb [hunk 1]
```diff
@@ -106,6 +106,11 @@ def self.mark_pending_bootstrap
                                            Minion.highstates[:failed]])
           .update_all highstate: Minion.highstates[:pending]
   end
+
+  # Forcefully updates all nodes to a pending highstate
+  def self.mark_pending_bootstrap!
+    Minion.update_all highstate: Minion.highstates[:pending]
```
Comment (ereslibre) [inline app/models/minion.rb]: There are some minions that might not have roles assinged (pending acceptance), we shouldn't update those, so this should be: `Minion.assigned_role.update_all highstate: Minion.highstates[:pending]`

**Review by ereslibre — CHANGES_REQUESTED** [review]

File: app/models/minion.rb (hunk 1 above)
Comment (MaximilianMeister) [reply app/models/minion.rb]: updated, thanks

**Review by vitoravelino — APPROVED** [review]

**Review by ereslibre — APPROVED** [review]
LGTM

**Pull request merged**